src/app/flashcard-editor/flashcard-editor.component.ts

```diff
@@ -96,4 +108,24 @@ export class FlashcardEditorComponent {
         'left': (window.innerWidth*.5) + rightWidth! - buttonWidth! - 16
     });
   }
+
+  uploadImage() {
+    let dialog = this.dialogRef.open(
+      UploadPopupComponent,
+      { data: this.selectedCard, disableClose: true}
+    )
+    dialog.afterClosed().subscribe(res => {
+      if(res.data.path != "") {
+        sessionStorage.setItem(res.data.path, res.data.data)
+        this.images.set(this.selectedCard.id, res.data.path)
```

Comment:
I get an exception if I try to upload the same image to two cards. I think this where the exception is taking place
![image](https://github.com/PonderTeam/ponder-app/assets/4361319/e24d7225-8f92-43e6-8236-b8ddd6691e24)

I think there are some use cases where a user may want to have duplicate images (like a labeled diagram).